### PR TITLE
#1 adding reserve and repeat functions.

### PR DESCRIPTION
**Pull request by evangilo**

### CODE REVIEW

**Comment by diasbruno** [comment]
Any ideas for a better name?

**Comment by evangilo** [comment]
Maybe `createList` where the first parameter could be the list size and  second could be the default value. What do you think about it?

**Comment by diasbruno** [comment]
Hmm, perhaps, we can use `reservedList(n)`, this will resemble with the c++ `std::vector<T>::reserve(N)`.

**Comment by evangilo** [comment]
I think that could use `reserve` instead `reserved`.

**Comment by diasbruno** [comment]
Ahh, `reserveWithDefault(n, i) = repeat(n, i)`.

File: readme.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ Simple list utilities.
...
 
 // split an 'ls' into chunk of 'size'.
+
+
+$ reserve(size : Number) -> [a]
+
+// returns an Array with N elements.
+
+
+$ repeat(value : Object, size : Number) -> [a]
```
Comment (diasbruno) [inline readme.md]: `size` should be the first argument so the can curry `repeat(n)(...)`.

Comment (diasbruno) [reply readme.md]: The type of `value` should be `a`.

Comment (evangilo) [reply readme.md]: done

**Pull request merged**